### PR TITLE
Feature: Change lesson buttons to clear style

### PR DESCRIPTION
**Pull request by Asartea**
## Because
- the current secondary buttons at the bottom of each lessons are both inaccessible and also distracting when placed directly next to a primary button ("Mark complete") in this case.

## This PR
- Changes the button style to `button--clear`

## Issue
Part of #3923 

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `keyword: brief description of change` format, using one of the following keywords:
  - `Feature` - adds new or amends existing user-facing behavior
  - `Chore` - changes that have no user-facing value, refactors, dependency bumps, etc
  - `Fix` - bug fixes
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] I have verified all tests and linters pass after making these changes.
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section

### CODE REVIEW

File: app/views/lessons/_lesson_buttons.html.erb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <div class="flex flex-col justify-center max-w-sm mx-auto space-y-6 md:max-w-full md:space-y-0 md:flex-row md:space-x-7">
-  <%= link_to path_course_path(course.path, course), class: 'button button--secondary font-medium', data: { test_id: 'view-course-btn' } do %>
+  <%= link_to path_course_path(course.path, course), class: 'button button--clear font-medium', data: { test_id: 'view-course-btn' } do %>
```
Comment (KevinMulhern) [inline app/views/lessons/_lesson_buttons.html.erb:2]: Nit: They look a tiny bit 🤏  too wide alongside the complete button. Reducing the padding sorts it out:

```suggestion
  <%= link_to path_course_path(course.path, course), class: 'button button--clear px-4 font-medium', data: { test_id: 'view-course-btn' } do %>
```

The next button will need this change too.

**Review by KevinMulhern — CHANGES_REQUESTED** [review]

File: app/views/lessons/_lesson_buttons.html.erb (hunk 1 above)
Comment (Asartea) [reply app/views/lessons/_lesson_buttons.html.erb:2]: Will do

**Comment by KevinMulhern** [comment]
Great find! yep I agree, for consistency and simplicity sake, having it the same for now would be best.

You're right though, it is a major action / milestone. I'd eventually like for us to do something different here.

**Comment by Asartea** [comment]
Done

**Review by KevinMulhern — APPROVED** [review]
LGTM, thanks @Asartea 💪

**Pull request merged**